XRTK-Core/Packages/com.xrtk.core/Definitions/Devices/IMixedRealityHandJointService.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.Providers.Controllers;
+
+namespace XRTK.Definitions.Devices
+{
+    /// <summary>
+    /// Mixed Reality Toolkit device definition, used to instantiate and manage a specific device / SDK
+    /// </summary>
+    public interface IMixedRealityHandJointService : IMixedRealityControllerDataProvider
+    {
+        /// <summary>
+        /// Get a game object following the hand joint.
+        /// </summary>
+        Transform RequestJointTransform(TrackedHandJoint joint, Handedness handedness);
```

Comment:
You shouldn't return a transform.
The MRTK has a TrackedPose that has a position and rotation, along with a comparable interface so that it's easy to compare them.

Transform comparatively comes with a lot of bulk.

Also, I recommend recommend returning the transform as an out param, and returning a bool to determine if the joint was successfully returned.

Reply:
I'd also recommend using the prefix `TryGet` if you use @provencher's recommendation with the bool and out param.